index.html

```diff
@@ -64,14 +65,14 @@ <h1>Bikes for Refugees</h1>
               Providing donated bikes and accessories to refugees and asylum
               seekers in Scotland.
             </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <button class="donate">Donate a bike today</button>
+            <button class="volunteer">Volunteer</button>
           </div>
 
           <div>
             <h2 class="heading-underline">Learn more</h2>
 
-            <div>
+            <div class="articleArea">
```

Comment:
For many of these `<div>` elements that have a class similar to `article`, have a look through these [semantic HTML elements](https://developer.mozilla.org/en-US/docs/Glossary/semantics#semantic_elements) and see if there is one that fits. It is very similar to the name already in these classes!